packages/split-layout/src/styles/vaadin-split-layout-base-styles.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/component-base/src/style-props.js';
+import { css } from 'lit';
+
+export const splitLayoutStyles = css`
+  :host {
+    display: flex;
+    overflow: hidden !important;
+    transform: translateZ(0);
```

Comment:
I don't remember exactly why these are needed.

Reply:
Apparently `translateZ(0)` was added as a fix for stacking in nested layouts https://github.com/vaadin/vaadin-split-layout/commit/8a479048bd562a1f8ba5549141127c7ddfc18ae0 to address https://github.com/vaadin/vaadin-split-layout/pull/19#issuecomment-262521655 but I'm unsure if it's still needed.